### PR TITLE
ASI Documentation: User deletes images in bulk from Media Gallery

### PR DESCRIPTION
**Pull request by lenaorobei**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/adobe-stock-integration/issues/1538

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs-beta.magento.com/user-guide/cms/media-gallery-asset-management.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cms/media-gallery-asset-management.md [hunk 1]
```diff
@@ -39,18 +39,18 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
     ![]({% link images/images/enhanced-media-gallery/selected-asset.png %}){: .zoom}
     _Add Selected_
 
-## Delete an asset
+## Delete assets
 
 1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.
 
-1. Select the asset, then click <span class="btn">Delete Selected</span>.
+1. Click <span class="btn">Delete Images...</span> and use checkboxes to select assets that need to be deleted.
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:46]:
```suggestion
1. Click <span class="btn">Delete Images...</span> and select the checkbox for each asset that you want to delete.
```

File: src/cms/media-gallery-asset-management.md [hunk 2]
```diff
@@ -39,18 +39,18 @@ Assets can be used extensively throughout the Admin, such as [add or edit a page
...
 
     ![]({% link images/images/enhanced-media-gallery/selected-asset.png %}){: .zoom}
```
Comment (leslietilling) [inline src/cms/media-gallery-asset-management.md:48]: It looks like the PR deletes this file -- did you want to include a replacement?

Comment (lenaorobei) [reply src/cms/media-gallery-asset-management.md:48]: Yes, it is included.
Screen Shot 2020-07-28 at 3 37 35 PM

**Comment by leslietilling** [comment]
running tests

**Pull request merged**